sites/pycharm-guide/contents/tutorials/intro-aws/setup/index.md

```diff
@@ -0,0 +1,114 @@
+---
+type: TutorialStep
+date: 2021-01-29
+title: Project Setup
+technologies: [AWS Toolkit]
+topics: [aws]
+author: mm
+subtitle: Install AWS SAM (Serverless Application Model), AWS CLI & finally PyCharm Plugin AWS Toolkit.
+thumbnail: ../thumbnail.png
+longVideo:
+  poster: poster_long.png
+  url: https://youtu.be/o4LdbOjarC0
+---
+
+In this tutorial step, let's install all the required dependencies.
+
+# AWS CLI
+Let's start first by downloading the AWS Command-Line Interface from `aws.amazon.com/cli`. 
+As I am using a Windows machine, I will download the 64-bit installer. 
+You can follow the same website if you are using Mac or Linux.
+
+![AWS CLI](./aws_cli.png)
+
+Before moving forward with AWS SAM, make sure you have installed below mentioned dependencies.
+  - AWS CLI 
+  - [Docker](https://www.docker.com/)
+
+
+# AWS SAM (Serverless Application Model)
+
+![AWS SAM](./aws_sam.png)
+
+According to Amazon, The AWS Serverless Application Model (SAM) is an open-source framework for building serverless applications.
+A serverless application is a combination of Lambda functions, 
+event sources, and other resources that work together to perform tasks.
+Note that a serverless application is more than just a 
+Lambda function—it can include additional resources such as APIs,
+databases, and event source mappings.
+
+You can download the setup file from `aws.amazon.com/serverless/sam` and follow the 
+same installation process like you did above for the CLI.
+
+
+# AWS Toolkit
+
+The AWS Toolkit for PyCharm is an open source plug-in for the PyCharm IDE
+that makes it easier to create, debug, and deploy Python applications on
+Amazon Web Services. The toolkit provides an integrated experience for 
+developing serverless applications, including assistance for getting 
+started, step-through debugging, and deploying from the IDE.
+
+If you are interested to know more about the plugin
```

Comment:
We should consider changing your style of mentioning links. Instead of saying "To learn more about this plugin, refer to this link" just make the first mention of "AWS Toolkit for PyCharm" into a link.